src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests_Global.cs

```diff
@@ -5875,8 +5875,7 @@ class H
             var model = compilation.GetSemanticModel(tree);
 
             var x1Decl = GetPatternDeclarations(tree, "x1").Single();
-            var x1 = (DeclarationPatternSyntax)x1Decl.Parent;
-            Assert.Null(model.GetAliasInfo(x1.Type));
+            Assert.True(x1Decl.Parent is VarPatternSyntax);
```

Comment:
I don't understand what this test is testing, aside that parsing the var syntax works. The semantic model doesn't seem to be used anymore.